uui-editor/src/plugins/colorPlugin/colorPlugin.tsx

```diff
@@ -33,8 +32,10 @@ interface IToolbarButton {
     editor: PlateEditor;
 }
 
+const modifiers = [{ name: 'offset', options: { offset: [0, 3] } }];
```

Comment:
Let's move this global variable into the component, because global variable assignment can be assumed as a side effect for bundlers, we should avoid using global scope variables, if possible

Reply:
It's not a global vaiable, it's inside the js module. I did it to avoid redundant use of `useMemo` to enable memoization

Reply:
yes, I understand, but for bundler each variable which created in the global scope(not inside the function or class) assumed like a side effect and if we have such things, we can't mark or package as `sideEffect: false`. But anyway Dropdown isn't wrapped in React.Memo, so it doesn't have any sense to wrap it into `useMemo` and also you still have inline arrow function in renderTarget prop, so this change doesn't make any optimizations ;)

Reply:
From my view nothing wrong with that code

Reply:
Discussed offline